src/Chapter1.hs

```diff
@@ -652,9 +660,12 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n
+	| num < 10 = num
+	| num > 10 = firstDigit (div num 10)
+	| otherwise = 1 -- Is 10
```

Comment:
This is right ⭐ 
You could also combine these two cases together, as in case of `10`, `div` would return you `1`, so you'll get the answer on the next iteration of the `firstDigit`'s recursive call 🤙🏼

Reply:
Thanks for the suggestion! Just implemented it